src/sentry/static/sentry/app/views/organizationDiscover/result/index.jsx

```diff
@@ -299,5 +305,4 @@ class Result extends React.Component {
   }
 }
 
-export {Result};
-export default withRouter(Result);
```

Comment:
I initially thought the bug was related to `withRouter` (I've had similar bugs due to it previously) - removed it just to be on the safe side